src/components/ProductCard/ProductCart.tsx

```diff
@@ -1,14 +1,58 @@
+import utils from '../../styles/modules/utils.module.css';
+import SelectedProduct from '../SelectedProduct/SelectedProduct';
+import StarIcon from '../Icon/StarIcon';
+import styles from './productCart.module.css';
+
 interface ProductCardProps {
   image: string;
   title: string;
   price: number;
   description: string;
   rating: number;
-  isActive: boolean;
+  ratingNumbers: number;
 }
 
-const ProductCard = (_: ProductCardProps) => {
-  return <></>;
+const formatRating = (rating: number): string => {
+  if (rating >= 1000000) {
+    return `${(rating / 1000000).toFixed(1)}m`;
+  } else if (rating >= 1000) {
+    // Adjust the logic to round to one decimal place and handle cases like 9999
+    const roundedRating = Math.floor(rating / 100) / 10; // Divide by 1000 and round to one decimal place
+    return `${roundedRating.toFixed(1)}k`;
+  } else {
+    return `${rating}`;
+  }
 };
```

Comment:
Split this into utils, rename for more generic not only format for rating

Reply:
fixed